packages/block-library/src/gallery/use-get-media.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import { some } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
+
+export default function useGetMedia( innerBlockImages ) {
+	const [ currentImageMedia, setCurrentImageMedia ] = useState( [] );
+
+	const imageMedia = useSelect(
+		( select ) => {
+			if (
+				! innerBlockImages?.length ||
+				some(
+					innerBlockImages,
+					( imageBlock ) => ! imageBlock.attributes.id
+				)
```

Comment:
This (and a few other spots) could use vanilla JS `Array.prototype.some()` instead of Lodash.

Reply:
I was just following some of the legacy code in the use of  lodash - happy to switch to vanilla js for these if this is the preferred option so have made this change.